assets/src/scss/pages/_page.scss

```diff
@@ -121,3 +121,57 @@
     outline: 0 none;
   }
 }
+
+.bottom-navigation {
+  height: 70px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-weight: 700;
+  font-family: var(--font-family-heading);
+
+  a:hover {
+    text-decoration: none;
+  }
+
+  &-next {
+    margin-left: auto;
+
+    &::after {
+      content: "\2192";
+      font-weight: bold;
+      margin-left: 7px;
```

Comment:
Can we use variables here? Maybe `$sp-1` which is 8px?

Reply:
Also, to make it simpler to accommodate for RTL sites too, you can use `margin-inline-end` or `margin-inline-start`